src/test/java/com/g4s8/ghman/data/DatabaseExtension.java

```diff
@@ -47,27 +45,19 @@
     private static final EmbeddedPostgres POSTGRES =
```

Comment:
@olenagerasimova does it have to be static to work correctly? I would prefer this to be tied to the test being run instead of the whole JVM